dask/array/tests/test_array_core.py

```diff
@@ -3906,6 +3919,14 @@ def test_to_zarr_delayed_creates_no_metadata():
 
 
 def test_zarr_existing_array():
+    zarr = pytest.importorskip("zarr")
+    a = zarr.array([1, 2, 3])
+    dsk = dict(da.from_zarr(a, inline_array=True).dask)
+    assert len(dsk) == 1
+    assert a in list(dsk.values())[0]
+
+
+def test_zarr_inline_array():
```

Comment:
I think the names of these tests are switched

```suggestion
def test_zarr_existing_array():
```